content/general/concepts/big-o-notation/big-o-notation.md

```diff
@@ -21,16 +21,16 @@ Here are some common runtimes:
-|   $O(1)$   |     Constant time (in purple)      | If the time needed by the algorithm is the same, regardless of the input size. | Accessing an element in an array.                                                                                                                                           |
+|  Runtime   |                Name                | Description                                                      | Use Case(s)                                                                                                                                                                 |
+| :--------: | :--------------------------------: | ---------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|  _O(n^2)_  | Polynomial/quadratic time (in red) | The time needed is the input size, `n`, multiplied by itself.    | [Bubble sort](https://www.codecademy.com/learn/sorting-algorithms-java/modules/bubble-sort-java/cheatsheet) has quadratic time complexity.                                  |
+|   _O(n)_   |       Linear time (in blue)        | The time needed is proportional to the input size `n`.           | Traversing an [array](https://www.codecademy.com/resources/docs/general/data-structures/array) of `n`-size.                                                                 |
+| _O(ln(n))_ |    Logarithmic time (in green)     | The time is a logarithmic function `ln()` of the input size `n`. | Analyzing the [binary search algorithm](https://www.codecademy.com/learn/fscp-22-search-graph-search-algorithms/modules/wdcp-22-binary-search-and-search-trees/cheatsheet). |
+|   _O(1)_   |     Constant time (in purple)      | The time needed is the same, regardless of the input size.       | Accessing an element in an array.                                                                                                                                           |
 
 ## Simplifying Big-O Expressions
 
-When determining an expression that characterizes the time or space complexity of an algorithm, an expression may contain multiple terms ( e.g., $O(n) + O(ln(n))$ ). In this situation, it is best practice to drop all of the lower order terms so the big-O notation would simplify into just O(n).
+When determining an expression that characterizes the time or space complexity of an algorithm, an expression may contain multiple terms ( e.g., _O(n) + O(ln(n))_ ). In this situation, it is best practice to drop all of the lower order terms so the big-O notation would simplify into just _O(n)_.
```

Comment:
Can we mention that we drop the lower-order terms because as *n* approaches infinity, the higher-order terms dominate and the lower-order terms become irrelevant?